### PR TITLE
docs: add FAQ about reuse kube-apiserver

### PR DESCRIPTION
**Pull request by pigletfly**
Signed-off-by: Wang Bing <wangbing.adam@gmail.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

add faq "install Karmada in a existing kubernetes cluster"

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
     NONE
```

### CODE REVIEW

File: docs/frequently-asked-questions.md [hunk 1]
```diff
@@ -20,3 +20,10 @@ to keep a consistent user experience and behavior.
 
 It's worth noting that not all controllers are needed by Karmada, for the recommended controllers please
 refer to [Recommended Controllers](./userguide/configure-controllers.md#recommended-controllers).
+
+
+## Can I install Karmada in a existing kubernetes cluster as host cluster?
```
Comment (RainbowMango) [inline docs/frequently-asked-questions.md:25]: I guess you want to explain `Can I install Karmada in an existing cluster and reuse the `kube-apiserver` as Karmada apiserver?` Am I right?

Comment (pigletfly) [reply docs/frequently-asked-questions.md:25]: yes

**Comment by RainbowMango** [comment]
I think it's worth noting. How about this?
```
## Can I install Karmada in a Kubernetes cluster and reuse the kube-apiserver as Karmada apiserver?

The quick answer is `yes`. In that case, you can save the effort to deploy
[karmada-apiserver](https://github.com/karmada-io/karmada/blob/master/artifacts/deploy/karmada-apiserver.yaml) and just
share the APIServer between Kubernetes and Karmada. In addition, the high availability capabilities in the origin clusters
can be inherited seamlessly. We do have some users using Karmada in this way.

There are some things you should consider before doing so:
- This approach hasn't been fully tested by the Karmada community and no plan for it yet.
- This approach will increase computation costs for the Karmada system. E.g.
  After you apply a `resource template`, take `Deployment` as an example, the `kube-controller` will create `Pods` for the
  Deployment and update the status persistently, Karmada system will reconcile these changes too, so there might be 
  conflicts.

TODO: Link to adoption use case once it gets on board.
```

**Comment by RainbowMango** [comment]
cc @pigletfly

**Review by RainbowMango — APPROVED** [review]
/retitle docs: add FAQ about reuse kube-apiserver
/lgtm
/approve

**Pull request merged**